ko/share-template-msgs.json

```diff
@@ -0,0 +1,14 @@
+{
+    "1": {
```

Comment:
This needs to be in Korean

Reply:
what matt means is the words that follow this. not the keys.

`"key": "translate this stuff"`